GiraffeApp/Program.fs

```diff
@@ -36,30 +36,53 @@ let pageHandler (title: string)(view: XmlNode list)(next: HttpFunc)(ctx: HttpCon
     ctx.SetHttpHeader ("Vary", "HX-Boosted")
     htmlNodes nodes next ctx
     
+let csrfHtmlNodesHandler (htmlNodes : AntiforgeryTokenSet -> XmlNode list) : HttpHandler =

```

Comment:
@surferjeff 
These 2 handlers is similar to what you implemented. I just add CSRF support to it.